app/src/data/dcNavLinks.ts

```diff
@@ -10,7 +10,7 @@ export const dcNavLinks = [
     text: "Collections",
   },
   {
-    href: `${DC_URL}/divisions`,
```

Comment:
The test "has links" in `app/src/components/navMenu/navMenu.test.tsx` passes but it shouldn't. The divisions expectation still has `DC_URL` in the test. It only passes because the `APP_ENV` env var is set to "development" by default, so then `DC_URL` is an empty string. Update the test and we should think of a test environment set of env var values.

Reply:
Same situation in `app/src/components/navMenu/mobileNavMenu.test.tsx`.